e2e_tests/pytest.ini

```diff
@@ -10,6 +10,7 @@ markers =
     e2e_cpu_2a: end to end CPU tests with two agents
     e2e_cpu_elastic: end to end CPU tests with elasticsearch for logging
     e2e_cpu_postgres: end to end CPU tests for testing basic database functionality
+    e2e_cpu_rbac: end to end ee rbac tests
```

Comment:
In retrospect I'm not entirely sure if putting the RBAC test code in OSS was the right call -- fixing an EE bug could require an OSS change to tests.

We have had future RBAC tests (and on the same vein slurm specific tests) that live in EE rather than this approach of skipping while in OSS

https://github.com/determined-ai/determined-ee/blob/b706aab77101d57baf996f9482c4e8767913c612/e2e_tests/pytest.ini#L25

I don't think it honestly matters too much but just a question if it makes sense to move these tests to EE or move the model registry tests to OSS or its fine to have this split.

Reply:
Ilia earlier advised to leave them here to minimize changes and make rebasing easier. Which is why this PR ended up going halfway (keeping them in OSS but making `test_groups.py` easy to move should we want to later). If you end up finding a stronger preference, I'm happy to either make the move or support you in it.